src/theme/input-menu.ts

```diff
@@ -13,8 +13,9 @@ export default (options: Required<ModuleOptions>) => {
       label: 'p-1.5 text-xs font-semibold text-gray-900 dark:text-white',
       separator: '-mx-1 my-1 h-px bg-gray-200 dark:bg-gray-800',
       item: ['group relative w-full flex items-center gap-1.5 p-1.5 text-sm select-none outline-none before:absolute before:z-[-1] before:inset-px before:rounded-md data-disabled:cursor-not-allowed data-disabled:opacity-75 text-gray-700 dark:text-gray-200 data-highlighted:text-gray-900 dark:data-highlighted:text-white data-highlighted:before:bg-gray-50 dark:data-highlighted:before:bg-gray-800/50', options.transitions && 'transition-colors before:transition-colors'],
```

Comment:
Why not adjust the `item`, `label`, etc. size like in https://github.com/benjamincanac/ui3/pull/133/files of like what we did in `DropdownMenu` or `ContextMenu`?

Reply:
I've just updated #133 so you can take the theme from https://github.com/benjamincanac/ui3/pull/133/files#diff-a711597ea735c5113ce82d5ba273c11b013c74a445a65e90df5b0bf4db86dbf2R26. Let me know if you want me to do it 😊